docs/getting-started/README.md

```diff
@@ -1,3 +1,29 @@
+---
+meta:
+  - name: "title"
+    content: Cronos | Crypto.org EVM Chain | Getting Started
+  - name: "description"
+    content: Learn how to setup nodes, different SDK modules and our all-in-one command-line interface ethermintd in this technical documentation.
+  - name: "og:title"
+    content: Cronos | Crypto.org EVM Chain | Getting Started
+  - name: "og:type"
+    content: Website
+  - name: "og:description"
+    content: Learn how to setup nodes, different SDK modules and our all-in-one command-line interface ethermintd in this technical documentation.
+  - name: "og-image"
+    content: https://cronos.crypto.org/og-image.png
+  - name: "twitter:title"
+    content: Cronos | Crypto.org EVM Chain | Getting Started
+  - name: "twitter:site"
+    content: "@cryptocom"
+  - name: "twitter:card"
+    content: summary_large_image
+  - name: "twitter:description"
+    content: Learn how to setup nodes, different SDK modules and our all-in-one command-line interface ethermintd in this technical documentation.
+  - name: "twitter-image"
+    content: https://cronos.crypto.org/og-image.png
+---
```

Comment:
According to this tutorial: https://www.adamdehaven.com/blog/how-to-add-metadata-canonical-urls-and-structured-data-to-your-vuepress-site/#add-the-canonical-url-to-every-page

We could use `canonicalUrl` in the yaml to specific a canonical link. (If we upgrade vuepress to v1.7.1+)

Such as `canonicalUrl: https://cronos.crypto.org/docs/getting-started/`.